scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -193,9 +193,9 @@
         {% endif %}
           <td>
               {% if compound.not_loaded %}
-                {{ compound.display_name }} <small>(not loaded)</small>
+                {{ compound.display_name|truncate(20, True) }} <small>(not loaded)</small>
```

Comment:
Only in these 2 first conditions the display name was not truncated. It truncated in the else, at line 204